packages/opencensus-web-exporter-ocagent/src/adapters.ts

```diff
@@ -29,7 +28,7 @@ const RECENT_EPOCH_MS = 1500000000000;  // July 13, 2017.
  * Converts a RootSpan type from @opencensus/core to the Span JSON structure
```

Comment:
Should we change this comment from ```@opencensus/core``` -> ```@opencensus/web-core```?

Reply:
Good catch, done.